doc/lsst.ap.pipe/apdb.rst

```diff
@@ -39,40 +38,40 @@ Should the user wish to use the pipeline on data containing bands not in the ``u
 
 .. prompt:: bash
 
    apdb-cli create-sql sqlite:///databases/apdb.db apdb_config.py
-   pipetask run -p ApPipe.yaml -c diaPipe:apdb.db_url="sqlite:///databases/apdb.db" differencer:coaddName=dcr -b repo -o myrun
-
-.. warning::
+   pipetask run -p ApPipe.yaml -c parameters:apdb_config=apdb_config.py differencer:coaddName=dcr -b repo -o myrun
 
-   Make sure the APDB is created with a configuration consistent with the one used by the pipeline.
-   Note that the pipeline file given by ``-p`` may include APDB config overrides of its own.
-   You can double-check what configuration is being run by calling :command:`pipetask run` with the ``--show config="apdb*"`` argument, though this lists *all* configuration options, including those left at their defaults.
-
-The ``apdb_config.py`` argument to |apdb-cli| specifies the name of the created configuration file that will contain serialized `~lsst.dax.apdb.ApdbConfig` for the new database.
-This file is not used yet by the ``pipetask`` options, but it will be used in the future.
+The ``apdb_config.py`` argument to |apdb-cli| specifies the name of the created configuration file that will contain a serialized `~lsst.dax.apdb.ApdbConfig` for the new database.
 
 A Postgres database can be set up and used with the following:
 
 .. prompt:: bash
     
    apdb-cli create-sql --namespace='my_apdb_name' 'postgresql://rubin@usdf-prompt-processing-dev.slac.stanford.edu/lsst-devl' apdb_config.py
-   pipetask run -p ApPipe.yaml -c diaPipe:apdb.db_url='postgresql://rubin@usdf-prompt-processing-dev.slac.stanford.edu/lsst-devl' -c diaPipe:apdb.namespace='my_apdb_name' -d "my_data_query" -b repo -i my/input/collection -o my/output/collection
+   pipetask run -p ApPipe.yaml -c parameters:apdb_config=apdb_config.py -d "my_data_query" -b repo -i my/input/collection -o my/output/collection
+
+If a pre-existing database is registered in the ``dax_apdb`` index, this becomes:
+
+.. prompt:: bash
+
+   pipetask run -p ApPipe.yaml -c parameters:apdb_config=label:db_name -d "my_data_query" -b repo -i my/input/collection -o my/output/collection
+
 
 A Postgres database can be set up and used within :ref:`bps yaml files <creating-a-yaml-file>` by adding this to a submit yaml:
 
 .. code-block:: yaml
 
-  extraQgraphOptions: "-c diaPipe:apdb.db_url='postgresql://rubin@usdf-prompt-processing-dev.slac.stanford.edu/lsst-devl' -c diaPipe:apdb.namespace='my_apdb_name'"
+  extraQgraphOptions: "-c parameters:apdb_config=/path/to/apdb_config.py"
 
 .. prompt:: bash
 
    apdb-cli create-sql --namespace='my_apdb_name' 'postgresql://rubin@usdf-prompt-processing-dev.slac.stanford.edu/lsst-devl' apdb_config.py
   
-Note that |apdb-cli| must be run with the same `namespace` prior to submitting this bps yaml.
+Note that |apdb-cli| must be run prior to submitting this bps yaml.
```

Comment:
For clarity, please specify that the `apdb_config.py` created by `apdb-cli` must be passed to diaPipe, as in the example above.